agentMET4FOF/metrological_agents.py

```diff
@@ -3,24 +3,14 @@
 import plotly.graph_objs as go
 from time_series_buffer import TimeSeriesBuffer
 from time_series_metadata.scheme import MetaData
-import numpy as np
 from agentMET4FOF.agents import AgentMET4FOF
 
 
 class MetrologicalAgent(AgentMET4FOF):
-    # dict like {
-    #     <from>: {
-    #         "buffer": TimeSeriesBuffer(maxlen=buffer_size),
-    #         "metadata": MetaData(**kwargs).metadata,
-    #     }
+
     _input_data: Dict[str, Dict[str, Union[TimeSeriesBuffer, Dict]]]
```

Comment:
I always liked this clarifications for the dicts. We could take them out, but it should serve better understanding to leave them in. Actually we should rather put them as docstrings below the attributes, so they get shown on ReadTheDocs. Still, this is just a suggestion. If you see clear reasons to take this out, I am fine with it.
```suggestion
    _input_data: Dict[str, Dict[str, Union[TimeSeriesBuffer, Dict]]]
    """Input dictionary of all incoming data including metadata
    
    dict like {
        <from>: {
            "buffer": TimeSeriesBuffer(maxlen=buffer_size),
            "metadata": MetaData(**kwargs).metadata,
        }
    """
```

Reply:
I should have taken a closer look. I thought it was old commented code.

Reply:
Happened to me earlier as well. If it was commented code, I would totally agree with you in just removing it. :+1: